conf/base.config

```diff
@@ -9,75 +9,40 @@
  * run on the logged in environment.
  */
 
-params {
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-}
-
-process {
+process{
 
-  cpus = { check_max( 1, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
+  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+  memory = { check_max( 7.GB * task.attempt, 'memory' ) }
+  time = { check_max( 4.h * task.attempt, 'time' ) }
 
-  errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'finish' }
+  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
   maxRetries = 1
   maxErrors = '-1'
 
   // Process-specific resource requirements
-  withName: "multiqc|get_software_versions|summary" {
-    memory = { check_max( 2.GB * task.attempt, 'memory' ) }
+  // NOTE - Only one of the labels below are used in the fastqc process in the main script.
```

Comment:
```suggestion
```

Reply:
Should we remove all the comments in base.config instead?

Reply:
I think you can remove this comments, you did the changes already